packages/snaps-utils/tsconfig.build.json

```diff
@@ -11,5 +11,10 @@
     "./src/**/test-utils",
     "./src/**/__mocks__",
     "./src/**/__snapshots__"
+  ],
+  "references": [
+    {
+      "path": "../snaps-ui/tsconfig.build.json"
```

Comment:
This means that `snaps-ui` will never be able to use any functions from `snaps-utils`. That may be fine for now, but could be come an issue in the future.